### ISSUE TITLE
Problem with error propagation?

### ISSUE THREAD

**Issue by valeriangalliat** [FWeinb/electron-screenshot-service#8]
I was rendering http://sassylists.com/documentation.html (domain doesn't exist anymore) and a popup shown me this error:

```
Uncaught Exception:
TypeError: Cannot read property 'id' of null
    at makeScreenshot (sassdoc.github.io/node_modules/electron-screenshot-service/electron-service/node_modules/electron-screenshot-app/index.js:58:44)
    at Timer.listOnTimeout (timers.js:89:15)
```

Also in the console output, I could log an error `{}` (literally an empty object).

So maybe there's an error at some level with error propagation?

**Comment by FWeinb** [comment]
Ugh. Thanks for reporting. There is a serious race condition in the code when the page is failing to load. I hope I will get some time to fix this.

**Comment by FWeinb** [comment]
Fixed in 2.0.0
